packages/api/models/nomination.js

```diff
@@ -144,10 +144,6 @@ module.exports = (sequelize, DataTypes) => {
         allowNull: true,
         type: DataTypes.DATE
       },
-      reminderSent: {
```

Comment:
Since we're removing this from the model I think we need a migration

Reply:
npx sequelize-cli db:migrate:undo --name 20210421232341-add_reminder_sent_boolean_nomination_column.js 
so we ran this to revert the migration, and then we deleted the migration file so when db:migrate runs it wouldn't add the migration.  Is that correct?

Reply:
That would work if this hadn't been deployed yet, but since that migration has already been run on the server, the only way to get rid of the column is to create a new migration that deletes the column. So we'll need to add that migration back and then create a new one to delete the column. In theory, we could manually run that command on the server as well, but usually it's best to avoid manually running commands on production servers if possible.

Reply:
Interesting.  Will fix that then no problem.

Reply:
Add the migration back and added another to remove the column.  Also updated the files to send email to health rep and provider rep (so mailer and nominations was altered).  Tested it by running npm run reminder-emails

Reply:
FYI, The way I went about adding the migration back was to do a hard reset on the last commit which was the commit that removed the migration file and then doing a push --force to GitHub. This resulted in returning back to a previous state, when the migration wasn't deleted.  Don't think that would cause problems but just wanted to mention it just in case it would.

Reply:
Fixed the 'aquire' typo brought up in the retrospective meeting as well